src/components/static-map.js

```diff
@@ -25,7 +25,7 @@ import PropTypes from 'prop-types';
 import {normalizeStyle} from '../utils/style-utils';
 
 import WebMercatorViewport from 'viewport-mercator-project';
-import AutoSizer from 'react-virtualized-auto-sizer';
+import ResizeObserver from 'resize-observer-polyfill';
```

Comment:
A traditional polyfill would typically patch the global `ResizeObserver`...

If this polyfill also works that way, would it make sense to leave this polyfill for the app to import? 

I am not sure but I suspect that we are already not including basic polyfills that are needed for things to work on IE/Edge anyway, so the user already has to install something?

Now this library is only 7.8kb https://bundlephobia.com/result?p=resize-observer-polyfill@1.5.1.

So, no strong opinion, just raising the question.

And of course, I realize that we are gaining almost as much as we lose by removing https://bundlephobia.com/result?p=react-virtualized-auto-sizer@1.0.2.

Reply:
- This library does not mutate the global object. It does export the native implementation if it exists.
- ResizeObserver is also [missing](https://caniuse.com/#search=resizeobserver) from some less common mobile browsers. I don't think we require polyfill in Edge legacy right now.